mkfs/rootdir.c

```diff
@@ -95,6 +95,7 @@ static struct rootdir_path current_path = {
 
 static struct btrfs_trans_handle *g_trans = NULL;
 static struct list_head *g_subvols;
+static u64 *g_default_subvol_id;
```

Comment:
Can't we just make it a u64, not a pointer to u64?

And since we have such a value, why not directly set it inside `btrfs_mkfs_fill_dir()` and we can avoid the extra `default_subvolid` parameter?